<ui version="4.0" stdsetdef="1" >
  <author>Ariya Hidayat &lt;ariya@kde.org></author>
  <comment>This file is part of the KDE project
Copyright (C) 2004 Ariya Hidayat &lt;ariya@kde.org>

This library is free software; you can redistribute it and/or
modify it under the terms of the GNU Library General Public
License as published by the Free Software Foundation; either
version 2 of the License, or (at your option) any later version.

This library is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU Library General Public License for more details.

You should have received a copy of the GNU Library General Public
License along with this library; see the file COPYING.LIB.  If not,
write to the Free Software Foundation, Inc., 51 Franklin Street,
Fifth Floor, Boston, MA 02110-1301, USA.</comment>
  <exportmacro></exportmacro>
  <class>KSpread::SheetPropertiesWidget</class>
  <widget class="QWidget" name="SheetPropertiesWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>408</width>
        <height>206</height>
      </rect>
    </property>
    <property name="sizePolicy" >
      <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="windowTitle" >
      <string>SheetPropertiesBase</string>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <layout class="QHBoxLayout" >
          <item>
            <layout class="QHBoxLayout" >
              <item>
                <widget class="QLabel" name="textLabel1" >
                  <property name="text" >
                    <string>Layout direction:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KComboBox" name="directionComboBox" >
                  <property name="whatsThis" stdset="0" >
                    <string>Defaut is that the first column of the sheet is on the left. If you choose Right to Left, then the first column will be on the right and the others added from right to left.</string>
                  </property>
                  <item>
                    <property name="text" >
                      <string>Left to Right</string>
                    </property>
                  </item>
                  <item>
                    <property name="text" >
                      <string>Right to Left</string>
                    </property>
                  </item>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="spacer2" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Fixed</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QGridLayout" >
          <item row="4" column="1" >
            <widget class="QCheckBox" name="showCommentIndicatorCheckBox" >
              <property name="text" >
                <string>Show c&amp;omment indicator</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If this box is checked cells containing comments will be marked by a small red triangle at the top right corner.</string>
              </property>
            </widget>
          </item>
          <item row="3" column="0" >
            <widget class="QCheckBox" name="useLCModeCheckBox" >
              <property name="text" >
                <string>Use &amp;LC mode</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If this box is checked the cell reference shown at the left end of the Formula Bar will be displayed in LC mode (i.e. L2C3) rather than in its normal form B3. This does not seem to be of much use at the moment.</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1" >
            <widget class="QCheckBox" name="showPageBordersCheckBox" >
              <property name="text" >
                <string>Show page &amp;borders</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If you check this option, the page borders will be drawn on your current sheet. Per default the page borders are not displayed. It is useful to see the page borders if you want to print your sheet.</string>
              </property>
            </widget>
          </item>
          <item row="4" column="0" >
            <widget class="QCheckBox" name="capitalizeFirstLetterCheckBox" >
              <property name="text" >
                <string>&amp;Convert first letter to uppercase</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Check this box and the first letter of any text you type in will automatically be converted to uppercase.</string>
              </property>
            </widget>
          </item>
          <item row="0" column="0" >
            <widget class="QCheckBox" name="hideZeroCheckBox" >
              <property name="text" >
                <string>&amp;Hide zero</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If this box is checked any cell containing the value zero will appear blank.</string>
              </property>
            </widget>
          </item>
          <item row="2" column="1" >
            <widget class="QCheckBox" name="showFormulaCheckBox" >
              <property name="text" >
                <string>Show &amp;formula</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If this box is checked KSpread will display the actual formulae in cells rather than the results.</string>
              </property>
            </widget>
          </item>
          <item row="2" column="0" >
            <widget class="QCheckBox" name="showColumnAsNumbersCheckBox" >
              <property name="text" >
                <string>Show column as &amp;numbers</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If this box is checked the column headings will show as numbers rather than as letters. Letters are default.</string>
              </property>
            </widget>
          </item>
          <item row="3" column="1" >
            <widget class="QCheckBox" name="showFormulaIndicatorCheckBox" >
              <property name="text" >
                <string>Show formula &amp;indicator</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If this box is checked KSpread will display a small Qt::blue triangle at the bottom left corner of cells containing formulae. This is useful if you want to protect cells with formulae.</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1" >
            <widget class="QCheckBox" name="showGridCheckBox" >
              <property name="text" >
                <string>Show &amp;grid</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If checked the grid (the cell limits) will be shown. This is default. If you uncheck it, the grid will be hidden.</string>
              </property>
            </widget>
          </item>
          <item row="1" column="0" >
            <widget class="QCheckBox" name="autoCalcCheckBox" >
              <property name="text" >
                <string>&amp;Automatic recalculation</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>This setting controls whether formulae are recalculated automatically when the value of any cell they refer to changes.</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="global">kdialog.h</include>
  </includes>
  <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header>kcombobox.h</header>
      <container>0</container>
      <pixmap></pixmap>
    </customwidget>
  </customwidgets>
</ui>
